runtime/altair/src/lib.rs

```diff
@@ -1489,16 +1489,14 @@ pub type UncheckedExtrinsic =
 /// Extrinsic type that has already been checked.
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 
-type UpgradeAltair1027 = pallet_loans_ref::migrations::v1::Migration<Runtime>;
-
 /// Executive: handles dispatch to the various modules.
 pub type Executive = frame_executive::Executive<
 	Runtime,
 	Block,
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	UpgradeAltair1027,
+	migrations::Migrations,
```

Comment:
Can we stay with the naming `UpgradeAltair1028` instead of tupling migrations here?

Reply:
I also like the specific info about the version the migration is referring to. The reason why I named it `Migrations` was so that we don't have to change the code for each RUT migration (i.e, have to update the version in the name) and instead have that info as a comment above the migration tuple. If you (and @wischli) still prefer to keep the version info in the name I am cool with that 👍  let me know